website/templates/docs.html

```diff
@@ -266,7 +267,7 @@ <h2 id=strings>Strings</h2>
 In V, a string is a read-only array of bytes. String data is encoded using UTF-8. 
 
 <p> 
-Strings are immutable. This means that the substring function is very efficient: 
+String contents are immutable. This means that the substring function is very efficient: 
```

Comment:
Nothing wrong with "strings are immutable"

Reply:
Strings have length, and the length is not immutable.

Reply:
Yes it is. It can't be modified.

Reply:
Oh, so we can't append to a string to change it's length?